WORKSPACE

```diff
@@ -227,12 +223,47 @@ load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
 
 node_repositories()
 
-register_toolchains("//toolchain:nodejs")
-
 # ----------------------------------------------------
 # proto_repositories
 # ----------------------------------------------------
 
 load("//:proto_repositories.bzl", "proto_repositories")
 
 proto_repositories()
+
+# ----------------------------------------------------
+# Typescript
+# ----------------------------------------------------
+load("@aspect_rules_ts//ts:repositories.bzl", "rules_ts_dependencies")
+load("@aspect_rules_ts//ts:repositories.bzl", "LATEST_VERSION")
+
+rules_ts_dependencies(
+    # This keeps the TypeScript version in-sync with the editor, which is typically best.
+    # ts_version_from = "//:package.json",
+
+    # Alternatively, you could pick a specific version, or use
+    ts_version = LATEST_VERSION,
+)
+
+# Fetch and register node, if you haven't already
+load("@rules_nodejs//nodejs:repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")
+
+nodejs_register_toolchains(
+    name = "node",
+    node_version = DEFAULT_NODE_VERSION,
+)
+
+load("@aspect_rules_js//npm:npm_import.bzl", "npm_translate_lock")
+
+npm_translate_lock(
```

Comment:
@pcj This part really should be part of `deps/ts_proto_deps.bzl`, as things are designed today, but I didn't think it was worth the complexity of updating the code generator. 

What do you think?